.github/workflows/sanity-test.yaml

```diff
@@ -0,0 +1,307 @@
+name: Sanity Testing
+on: [push]
```

Comment:
We should add `workflow_dispacth` before we merge.

Reply:
Yes. And as discussed with Vaibhav, will add - run on merge to main